react-emoji/src/components/EmojiContainer.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import EmojiRow from './EmojiRow';
+import emoji from '../emoji.json';
+import './EmojiContainer.css'
+
+const emojiList = emoji;
+
+export default function EmojiContainer(props){
+    const {searchEmoji} = props;
+    const emojiBlock = emojiList.filter((item => {
+        if(item.title.includes(searchEmoji) || item.keywords.includes(searchEmoji)){
+            return item
+        }
+    })).slice(0,15).map((item) => <EmojiRow  item={item} key={item.symbol} /> )
```

Comment:
could be simplified:
```suggestion
    const emojiBlock = emojiList
      .filter(item => item.title.includes(searchEmoji) || item.keywords.includes(searchEmoji))
      .slice(0,15)
      .map((item) => <EmojiRow  item={item} key={item.symbol} /> )
```